static/app/components/events/featureFlags/eventFeatureFlagList.spec.tsx

```diff
@@ -208,8 +217,25 @@ describe('EventFeatureFlagList', function () {
     ).toBeInTheDocument();
   });
 
-  it('renders nothing if event.contexts.flags is not set', function () {
-    render(<EventFeatureFlagList {...NO_FLAG_CONTEXT_SECTION_PROPS} />);
+  it('renders cta if event.contexts.flags is not set and should show cta', async function () {
+    render(<EventFeatureFlagList {...NO_FLAG_CONTEXT_SECTION_PROPS_CTA} />);
+    screen.logTestingPlaygroundURL();
+
+    const control = screen.queryByRole('button', {name: 'Sort Flags'});
+    expect(control).not.toBeInTheDocument();
+    const search = screen.queryByRole('button', {name: 'Open Feature Flag Search'});
+    expect(search).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', {name: 'Set Up Integration'})
+    ).not.toBeInTheDocument();
+
+    // wait for the CTA to be rendered
+    expect(await screen.findByText('Feature Flags')).toBeInTheDocument();
```

Comment:
Should we make this `Set Up Feature Flags` in case we have a `Feature Flags` title somewhere can mean the below getByText can not exist.